app/views/people/show.html.erb

```diff
@@ -34,7 +34,7 @@
     } %>
   </div>
 
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds" dir="<%=page_text_direction%>">
```

Comment:
Rather than moving it up the DOM it's better to add it to the specific areas that need it, even if that means adding the `dir` attribute to a larger number of elements.